app/views/shared/_past_bookings.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="past-sessions">
+<% @past_bookings.each do |booking| %>
+    <div class="past-booking-card">
+      <!-- if Time.now <= @booking.start_time + 24hours -->
+      <div class="therapist-avatar d-flex flex-column" style="padding: 10px 0 0 10px">
```

Comment:
NO INLINE STYLE AARRG !!! lol (*nice voice* can you please put this in the proper css component file ?)

Reply:
Or use bootstrap classes ! equally okay !